src/gui/threejsScene.js

```diff
@@ -549,11 +549,6 @@ import { ViewFrustum, frustumVertexShader, frustumFragmentShader, MAX_VIEW_FRUST
 
         raycaster.firstHitOnly = true; // faster (using three-mesh-bvh)
 
-        // add object layer to raycast layer mask
-        objectsToCheck.forEach(obj => {
-            raycaster.layers.mask = raycaster.layers.mask | obj.layers.mask;
-        });
-
```

Comment:
Is there a reason this is removed? I remember adding it for some reason but now I forget exactly why.

Reply:
It's removed because there is another line right below where the exact same piece of code is run redundantly.

Reply:
Good catch! 🕵️